BlocQuery/BQUser.m

```diff
@@ -30,6 +30,31 @@ + (void)load
     
 }
 
+- (PFFile*)setDefaultProfileImage
```

Comment:
I would rename this method:
```
+ (PFFile*)defaultProfileImage
```
As it is I'd expect this method to take a parameter (a `PFFile`) and set that image as the new default.